src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml.cs

```diff
@@ -79,31 +76,27 @@ private async Task Initialize()
                 Status.Text = "Loading Utility Network...";
 
                 // Create a map.
-                MyMapView.Map = new Map(BasemapStyle.ArcGISDarkGray)
+                MyMapView.Map = new Map(new Uri("https://sampleserver7.arcgisonline.com/portal/home/item.html?id=be0e4637620a453584118107931f718b"))
                 {
                     InitialViewpoint = _startingViewpoint
                 };
 
-                // Create and load a service geodatabase that matches utility network.
-                ServiceGeodatabase serviceGeodatabase = await ServiceGeodatabase.CreateAsync(new Uri(FeatureServiceUrl));
+                await MyMapView.Map.LoadAsync();
+
+                _utilityNetwork = MyMapView.Map.UtilityNetworks.FirstOrDefault();
+                await _utilityNetwork.LoadAsync();
+
+                var serviceGeodatabase = _utilityNetwork.ServiceGeodatabase;
 
                 // Add the layer with electric distribution lines.
```

Comment:
Can we replace code comment with 

```suggestion
                // Get the layer with electric distribution lines.
```